### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by asifaminisonline**
### About

- [ ] Added javascript validation to the contact form.
- [ ] Followed GitHub flow.
- [ ] No linter errors.

### CODE REVIEW

File: validation.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+   inputFeedbackEl.innerText = 'Looks Good';
+};
+
+let makeInValid = (inputEl,inputFeedbackEl) =>{
+    inputEl.classList.remove('is-form-valid');
+    inputEl.classList.add('is-form-invalid');
+    inputFeedbackEl.classList.remove('text-success');
+    inputFeedbackEl.classList.add('text-danger'); 
+    inputFeedbackEl.innerText = `Please Enter ${inputEl.placeholder}`;
+ };
```
Comment (codecaiine) [inline validation.js]: - [ ] Good job so far. From the project requirement when the user submits the form, if the email is in **lowercase** the validation is OK then the form is sent. In your case even when it is lowercase the form is not sent. Please fix it in order to make your form work as expected  :+1: 

- [ ] According to the project requirements when the email is not **lowercase** the validation is not OK, and an error message shows up to the user near the submit button informing them of the error and the form is not sent. Please fix it in order to make your form work as expected :+1: 
- [ ] Please make sure to fix your **Textarea** message requirement. We can't submit the form info :smile: 


Screenshot from 2022-11-17 21-21-19

File: validation.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+.text-danger {
+  padding-left: 10px;
+  color: darkred;
+  font-size: 15px;
+  font-weight: 200;
+  position: relative;
+  bottom: 22px;
+}
```
Comment (codecaiine) [inline validation.css]: - [ ] Please make sure that your contact form stick to the design guidelines (e.g., font, colors, margins) of [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as required for this project you can see a bit differences between your design and the chosen Figma :+1: 

#### Your design
Screenshot from 2022-11-17 21-36-17

#### Figma design
contact_form

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @asifaminisonline @Besufekad-HAZ , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good use of github flow :heavy_check_mark: 

## Required Changes ♻️

- [ ] I noticed that you set up the wrong linter config . Please make sure to use The [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) and set them correctly in this project repo. :+1: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mckent05 — APPROVED** [review]
Hi @asifaminisonline,

# STATUS: APPROVED ✔️ ✔️  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Contact form validation works as expected 👍 
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**